app/com/kinja/play/plugins/Closure.scala

```diff
@@ -45,7 +43,7 @@ class InvalidClosureValueException(obj: Any, path: Option[String] = None) extend
 /**
  * Play plugin for Closure.
  */
-class ClosurePlugin(app: Application) extends Plugin {
+class ClosurePlugin @Inject() (implicit app: Application) extends Plugin {
```

Comment:
First, I won't use any dependency injection, as there's a lot to change without introducing DI, so this was the easiest way to start.
In upcoming PRs we can upgrade mantle to use dependency injection, but first it's easier to do the minimum, as there is still a lot regarding soy-related updates.